content/en/blog/_posts/2022-05-03-kubernetes-release-1.24.md

```diff
@@ -0,0 +1,237 @@
+* [Pod Affinity NamespaceSelector](https://github.com/kubernetes/enhancements/issues/2249): Add a `namespaceSelector` field for to pod affinity/anti-affinity spec.
+* [Leader Migration for Controller Managers](https://github.com/kubernetes/enhancements/issues/2436): kube-controller-manager and cloud-controller-manager can apply new controller-to-controller-manager assignment in HA control plane without downtime.
+* [CSR Duration](https://github.com/kubernetes/enhancements/issues/2784): Extend the CertificateSigningRequest API with a mechanism to allow clients to request a specific duration for the issued certificate.
+
+### Major Changes
+
+This release saw two major changes: 
+
+* [Dockershim Removal](https://github.com/kubernetes/enhancements/issues/2221)
+* [Beta APIs are off by Default](https://github.com/kubernetes/enhancements/issues/3136)
+
+### Release Notes
+
+Check out the full details of the Kubernetes 1.24 release in our [release notes](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.24.md).
+    
+### Availability
+
+Kubernetes 1.24 is available for download on [GitHub](https://github.com/kubernetes/kubernetes/releases/tag/v1.24.0).
+To get started with Kubernetes, check out these [interactive tutorials](/docs/tutorials/) or run local 
+Kubernetes clusters using containers as “nodes”, with [kind](https://kind.sigs.k8s.io/).
+You can also easily install 1.24 using [kubeadm](/docs/setup/independent/create-cluster-kubeadm/).
+
+### Release Team
+
+This release would not have been possible without the combined efforts of committed individuals 
+comprising the Kubernetes 1.24 release team. This team came together to deliver all of the components 
+that go into each Kubernetes release, including code, documentation, release notes, and more. 
+
+Special thanks to James Laverack, our release lead, for guiding us through a successful release cycle, 
+and to all of the members of the release team for the time and effort they put in to deliver the v1.24 
+release for the Kubernetes community. 
+
+### Release Theme and Logo 
+
+**Kubernetes 1.24: Stargazer**
+
+{{< figure src="/images/blog/2022-05-03-kubernetes-release-1.24/kubernetes-1.24.png" alt="" class="release-logo" >}}
+
+The theme for Kubernetes 1.24 is _Stargazer_.
+
+Generations of people have looked to the stars in awe and wonder, from ancient astronomers to the 
+scientists who built the James Webb Space Telescope. The stars have inspired us, set our imagination 
+alight, and guided us through long nights on difficult seas.
+
+With this release we gaze upwards, to what is possible when our community comes together. Kubernetes 
+is the work of hundreds of contributors across the globe, and thousands of end-users supporting 
+applications that serve millions. Every one is a star in our sky, helping us chart our course.
```

Comment:
This may be a style choice, but i think it should be:
```suggestion
applications that serve millions. Everyone is a star in our sky, helping us chart our course.
```